xarray/core/utils.py

```diff
@@ -18,6 +18,16 @@
     OrderedDict, basestring, bytes_type, dask_array_type, iteritems)
 
 
+def _check_inplace(inplace, default=False):
+    if inplace is None:
+        inplace = default
+    else:
+        warnings.warn('The inplace argument has been deprecated and will be '
+                      'removed in xarray 0.12.0.', FutureWarning)
```

Comment:
Can you add the `stacklevel` argument with an appropriate value? (you'll need to experiment a bit)

That ensures that warnings end up pointing to the specific line of code that need to be updated.

Reply:
`stacklevel=3` seems to give useful warnings with the test suite

```
xarray/tests/test_dataset.py::TestDataset::()::test_expand_dims_error
  /home/deepak/work/python/xarray/xarray/tests/test_dataset.py:2016: FutureWarning: The inplace argument has been deprecated and will be removed in xarray 0.12.0.
    original.set_coords('z', inplace=True)
```